server/command.go

```diff
@@ -213,7 +220,10 @@ func (p *Plugin) runPopCommand(args []string, extra *model.CommandArgs) (bool, e
 	if foreignID != "" {
 		message := fmt.Sprintf("@%s popped a Todo you sent: %s", userName, issue.Message)
 		p.sendRefreshEvent(foreignID)
-		p.PostBotDM(foreignID, message)
+		err = p.PostBotDM(foreignID, message)
```

Comment:
Would you please also change this method to not return an error similar to `PostBotCustomDM`

Reply:
Refactored because it was sharing a lot of logic with PostBotCustomDM